portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/frontenddataset/datamanagment/FDSEntry.testcase

```diff
@@ -191,6 +191,10 @@ definition {
 			AssertElementPresent(locator1 = "ObjectPortlet#DELETE_ENTRY_BUTTON");
 
 			AssertElementPresent(locator1 = "FrontendDataSetAdmin#VIEW_ENTRY_BUTTON");
+
+			Click(
+				key_entry = "Test Dataset",
+				locator1 = "ObjectPortlet#ENTRY_KEBAB_MENU");
```

Comment:
Hey @abhnerramos I don't understand how a click after the asserts can stabilize this test, what was your idea? 🤔

Reply:
The last CI action on test is to open the entry kebab menu (the dropdown) and it stays opened. Since that is the last code on the test, it goes to tearDown. On tearDown the first action is to click on the entry kebab menu to open the dropdown and delete the data set, but since it is already opened, it ends up closing the dropdown and the tearDown can't proceed to delete the data set.